sw/device/silicon_creator/rom/rom.c

```diff
@@ -96,6 +96,25 @@ static inline rom_error_t rom_irq_error(void) {
   return kErrorInterrupt + mcause;
 }
 
+/**
+ * Prints a status message indicating that the ROM is entering bootstrap mode.
+ */
+static void rom_bootstrap_message(void) {
+  uart_putchar('b');
+  uart_putchar('o');
+  uart_putchar('o');
+  uart_putchar('t');
+  uart_putchar('s');
+  uart_putchar('t');
+  uart_putchar('r');
+  uart_putchar('a');
+  uart_putchar('p');
+  uart_putchar(':');
+  uart_putchar('1');
+  uart_putchar('\r');
+  uart_putchar('\n');
+}
+
```

Comment:
What about about sending a `READ_STATUS` (`0x05`) and expecting `0x00` instead? It's configured [here]([#14450](https://github.com/lowRISC/opentitan/issues/14450)) if not we would expect 0xff on CIPO.

I appreciate the convenience of seeing a message on the console but IMHO isolating individual functionality could result in more informative test cases.

Reply:
I've updated the test case in the hjson file to include reporting this message over the uart.